app/services/track_commission_changes_service.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+class TrackCommissionChangesService
+  include SolanaLogic
+
+  def initialize(current_batch: , network: "mainnet", solana_url: nil)
+    @current_batch = current_batch
+    @previous_batch = Batch.where("created_at < ?", @current_batch.created_at)
+                           .where(network: network)
+                           .where.not(scored_at: nil)
+                           .last
+    @network = network
+    @solana_url = solana_url || NETWORK_URLS[@network]
+  end
+
+  def call
+    VoteAccount.includes(:validator).where(network: @network).in_batches(of: 100) do |batch|
```

Comment:
`batch` in that case seems misleading. Maybe just `va` / `vote_account`

Reply:
Why is it misleading? it is a batch of records